CONTRIBUTING.md

```diff
@@ -8,11 +8,20 @@ This project uses check-style plugins. Run some checks before you create a new p
 dev/reformat
 ```
 
+If you have changed code for Web UI, Run following command before you create a new pull request.
+
+```shell
```

Comment:
Could the frontend code format introduce util in `dev/reformat`?

Reply:
Now. When executing `mvn install`, the code will be formatted.

Reply:
@tiny-dust, could you update the `reformat` shell to provide developer with the way that reformats the frontend code and update the shell here?

Reply:
ok